sklearn/cross_decomposition/_pls.py

```diff
@@ -634,6 +634,8 @@ class PLSRegression(_PLS):
     In french but still a reference:
     Tenenhaus, M. (1998). La regression PLS: theorie et pratique. Paris:
     Editions Technic.
+
+    .. versionadded:: 0.8
```

Comment:
These need to be added after the description, before parameters. Adding it here will be a problem with numpydoc